corehq/pillows/mappings/utils.py

```diff
@@ -11,3 +12,62 @@ def mapping_from_json(filename):
                 .replace('"__DATE_FORMATS_ARR__"', json.dumps(DATE_FORMATS_ARR)))
         mapping = json.loads(data)
     return mapping
+
+
+def transform_for_es7(original_mapping):
```

Comment:
Is the aim here to replace the mapping file in a later PR after the migration?  It'd be a weird state to be in where we keep the mapping file in an out-of-date syntax and dynamically translate it to the "real" mapping.

Reply:
Yes, once all envs are migrated, we just update the mapping files to the latest syntax and remove this tranformation